_data/conferences.yml

```diff
@@ -81,3 +81,38 @@
   speaker: Lori Lalonde
   location: Kitchener, ON
   topic: Windows 10 Developer Camp
+
+- date: 2015-10-19
+  event: Winnipeg .NET User Group
+  link: http://www.winnipegdotnet.org
+  speaker: James Chambers
+  location: Winnipeg, MB
+  topic: What to Expect with MVC 6
+
+- date: 2015-11-9
+  event: Microsoft Virtual Academy
+  link: http://www.microsoftvirtualacademy.com
+  speaker: James Chambers
+  location: Seattle, WA
+  topic: Applications on Azure: Putting all the Pieces Together
```

Comment:
Include quotes around the topic titles. The colon will mess things up otherwise.